src/_includes/install/file-system-perms-before.md

```diff
@@ -5,7 +5,7 @@ This topic discusses how to set read-write permissions for the web server group
 The procedure you use is different, depending on whether you use [shared hosting](#perms-shared) and have one user or if you use a [private server](#perms-private) and have two users.
 
 {:.bs-callout-info}
-If you're using a Magento version*earlier than* 2.0.6, see [Appendix&mdash;Magento file system ownership and appendix (legacy)]({{ page.baseurl }}/install-gde/install/legacy-file-system-perms.html) instead.
+If you're using a Magento version*earlier than* 2.0.6, see [Appendix---Magento file system ownership and appendix (legacy)]({{ page.baseurl }}/install-gde/install/legacy-file-system-perms.html) instead.
```

Comment:
In a separate PR, remove this callout and delete the target file (legacy-file-system-perms.md) from the v2.3 directory. Also delete all links to that file, including those in src/_includes/install/file-system-perms-before_22.md, src/_includes/install/file-system-umask-over.md, and others

Reply:
PR: https://github.com/magento/devdocs/pull/6364

I hope I did all what you requested.